src/store/wizardSlice.ts

```diff
@@ -82,6 +91,13 @@ const initialState: wizardState = {
     shareMethod: 'sources',
     source: undefined,
   },
+  azure: {
+    tenantId: '',
+    shareMethod: 'sources',
+    subscriptionId: '',
+    source: '',
+    resourceGroup: '',
+  },
```

Comment:
Same here, I think it looks neater in dev tools :sweat_smile: 

```suggestion
  azure: {
    shareMethod: 'sources',
    tenantId: '',
    subscriptionId: '',
    source: '',
    resourceGroup: '',
  },
```